library/dokku_network_property.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# from ansible.module_utils.basic import *
+from ansible.module_utils.basic import AnsibleModule
+import subprocess
+
+DOCUMENTATION = """
+---
+module: dokku_network_property
+short_description: Set or clear a network property for a given dokku application
+options:
+  app:
+    description:
+      - The name of the app
+    required: True
+    default: null
+    aliases: []
+  network:
+    description:
+      - The name of the network to attach to
+    required: True
+    default: null
+    aliases: []
+  property:
+    description:
+      - The attach network property
+    required: True
+    default: null
+    choices: [ "initial-network", "attach-post-create", "attach-post-deploy" ]
+    aliases: []
+author: Philipp Sessler
+requirements: [ ]
+"""
+
+EXAMPLES = """
+- name: Associates a network after a container is created but before it is started
+  dokku_network_property:
+    app: hello-world
+    network: example-network
+    property: attach-post-create
```

Comment:
let's move the `property` above the value since, conceptually, you first need to know which property you are setting